pex/expose_scripts.py

```diff
@@ -0,0 +1,70 @@
+import hashlib
+import itertools
+import os
+
+from pathlib import Path
+
+
+SCRIPT_TEMPLATE = """#!/bin/sh
```

Comment:
This is not ideal. For example, it would fail in any Python [distroless](https://github.com/GoogleContainerTools/distroless) image whereas the corresponding python console script it wraps would not.